common/conf.py

```diff
@@ -290,42 +290,40 @@ def get_previous_version(majornumber, minornumber=0):
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'projstandardsdoc'
 
-
 # -- Options for LaTeX output --------------------------------------------------
 
-latex_elements = {
-    'classoptions': ',oneside',
-    'pointsize': '10pt',
-    'inputenc': '%% Unused',
-    'utf8extra': '%% Unused',
-    'fontenc' : '%% Unused',
-    'fontpkg': '%% Unused',
-    'babel': '%% Unused',
-    'printindex': '''\\phantomsection
-\\addcontentsline{toc}{part}{\indexname}
-\\printindex''',
-    'preamble': '''
-\input{../../../common/preamble.tex}
-''',
-}
+#latex_elements = {
+#    'classoptions': ',oneside',
+#    'pointsize': '10pt',
+#    'inputenc': '%% Unused',
+#    'utf8extra': '%% Unused',
+#    'fontenc' : '%% Unused',
+#    'fontpkg': '%% Unused',
+#    'babel': '%% Unused',
+#    'printindex': '''\\phantomsection
+# \\addcontentsline{toc}{part}{\indexname}
+# \\printindex''',
+#    'preamble': '''
+# \input{../../../common/preamble.tex}
+# ''',
+# }
 
 # For "manual" documents, if this is true, then toplevel headings are parts,
 # not chapters.
-latex_use_parts = True
+# latex_use_parts = True
```

Comment:
You don't need a space after (for consistency with other settings here, and in the other repos), same for the setting below